charts/mattermost-push-proxy/README.md

```diff
@@ -0,0 +1,124 @@
+Mattermost Push Proxy Helm Chart
+====================================================
+
+This is the Helm chart for the Mattermost Push Proxy. It is subject to changes, but is used by Mattermost internally by our [community.mattermost.com](https://community.mattermost.com) server to push notifications. To learn more about Helm charts, [see the Helm docs](https://helm.sh/docs/).
+
+To read an overview of how to migrate Mattermost to Kubernetes, [see this blog post](https://mattermost.com/blog/mattermost-kubernetes/).
+
+The Mattermost Team Edition Helm chart can be found [here](https://github.com/helm/charts/tree/master/stable/mattermost-team-edition).
+
+To install the Mattermost Team Edition Helm chart in a GitLab Helm chart deployment, [see this documentation](https://docs.mattermost.com/install/install-mmte-helm-gitlab-helm.html).
```

Comment:
I think instead of mentioning the blog post and the TE chart it's probably better to point at the push proxy repo to let readers know what the push proxy is

Reply:
@jwilander thanks for the review. I added the push proxy developers page and information about the Mattermost hosted push proxy solution.